jmdaemon/jmdaemon/onionmc.py

```diff
@@ -1435,10 +1435,10 @@ def wait_for_directories(self) -> None:
         # so we are guaranteed to have only directory peers.
         if len(self.get_connected_directory_peers()) < len(self.peers):
             self.directory_wait_counter += 1
-            # < 2*11 = 22 seconds; compare with CONNECT_TO_ONION_TIMEOUT;
+            # < 2*21 = 42 seconds; compare with CONNECT_TO_ONION_TIMEOUT;
             # with current vals, we get to try twice before entirely
             # giving up.
-            if self.directory_wait_counter < 11:
+            if self.directory_wait_counter < 21:

```

Comment:
Why not `CONNECT_TO_ONION_TIMEOUT / 2 + 1` instead of hardcoded `21`?

Reply:
Good idea, thanks :)

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1385/commits/8ab2d50a42c8fae2452deae89ae8c905af8677e3